backend/routes/userSession.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const session = require('express-session');
+const {PrismaSessionStore} = require('@quixo3/prisma-session-store')
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+const userSessionRouter = express.Router();
+
+const userSession = (session({
+    secret: process.env.SESSION_SECRET,
+    saveUninitialized: false,
+    resave: false,
+    cookie: {
+        secure: false,
+        httpOnly: true,
+        maxAge: 1000 * 60 * 30
```

Comment:
nit: extract the 1000 * 60 * 30 into a variable that explains the time you are trying to achieve such as THREE_DAYS. Better for readability.